quantities/serialization/Injectors.cs

```diff
@@ -0,0 +1,56 @@
+using Quantities.Measures;
+
+namespace Quantities.Serialization;
+
+internal sealed class ScalarInjector : IInject
+{
+    public IBuilder Inject<TMeasure>() where TMeasure : IMeasure => new Builder<TMeasure>();
+    public IBuilder Inject<TMeasure, TAlias>()
+        where TMeasure : IMeasure
+        where TAlias : IInjector, new() => new AliasedBuilder<TMeasure, TAlias>();
+}
+
+internal sealed class PowerInjector<TDim> : IInject
+    where TDim : IDimension
+{
+    public IBuilder Inject<TMeasure>() where TMeasure : IMeasure => Inject<TMeasure, Linear<TMeasure>>();
+    public IBuilder Inject<TMeasure, TAlias>()
+        where TMeasure : IMeasure
+        where TAlias : IInjector, new() => new AliasedBuilder<Power<TDim, TMeasure>, TAlias>();
+}
+
+internal sealed class FractionInjector : IInject
+{
+    public IBuilder Inject<TMeasure>() where TMeasure : IMeasure => new Nominator<TMeasure>();
+    public IBuilder Inject<TMeasure, TAlias>() where TMeasure : IMeasure where TAlias : IInjector, new() => throw new KeepUnusedException(this);
```

Comment:
How about `ShouldBeUnusedException`? This way, you better explain why you're throwing this exception.  
When I first read _KeepUnused_ I thought you were throwing because it is used and I had to go to the implementation.

Reply:
Thanks! A much better name. Done :-)